static/docs/user-guide/data-sharing/mounted-storage.md

```diff
@@ -0,0 +1,115 @@
+# Sharing Data Through a Mounted DVC Storage
```

Comment:
again, see my other comments. There are at least two possibilities - shared cache or shared remote. In case of NAS it's actually beneficial to share cache (and use some regular cloud remote to still do backups).

Reply:
> In case of NAS it's actually beneficial to share cache (and use some regular cloud remote to still do backups).

Sharing cache in the case of a NAS may cause problems when we try to use `dvc gc`. I remember seeing some discussions about this.

Reply:
Yes, and we even introduced a special flag - to pass multiple projects at once to `dvc gc`. Gc in DVC is a big pain still but it does not change the fact I mentioned above.

Reply:
> we even introduced a special flag - to pass multiple projects at once to `dvc gc`

The option `-p, --projects` of [`dvc gc`](https://dvc.org/doc/command-reference/gc) gets a path to a project (at least this is how I understand the man page, I have never tried it).

In the case of a NAS mounted storage I assume that the collaborating projects are located on different machines, isn't it? So, the option `-p, --projects` cannot be used in this case.

Reply:
There are probably ways still to run GC - clone projects to a single machine. It's not ideal (all about GC not) but it's a maintenance operations vs day to day workflow that is being optimized with links if you share cache vs sharing remote directly.

Also, I think it is the same problem with other your cases and in one of them it's about people sharing the same machine, right?

Reply:
> It's not ideal (all about GC not) but it's a maintenance operations vs day to day workflow that is being optimized with links if you share cache vs sharing remote directly.

I agree that if we disregard any issues about GC, then sharing the cache would indeed optimize data sharing for the NAS case and would require less storage (by using symlinks).
I may extend the tutorial and the user-guide page to explain this optimization as well.

> Also, I think it is the same problem with other your cases and in one of them it's about people sharing the same machine, right?

I don't think it applies to the other cases because it does not bring any improvements (in terms of optimization: faster and smaller disk space).

Reply:
> I may extend the tutorial and the user-guide page to explain this optimization as well.

I first understand the options in terms of organizing data, understand which of them are more general then others, then would try to come up with a couple of sections that explain them in a general way. And by general I mean concepts like -  cache is shared or not? people use a single machine or not? etc

Reply:
I don't see how my initial concern is resolved or addressed here. Please 🙏 , don't resolve them on your own - it makes it extremely hard to do reviews (check and follow up the previously raised concerns).

Reply:
To be precise - I don't see much value in three (two?) sections that explain different variations of the mounted remote. And to be even more precise - I haven't see a mounted share remote case. The only benefit I see - unsupported storage type. I would just create a How to or FAQ or something with one-two paragraphs explanation on options - mount, use rsync/rclone, etc.

Reply:
> I don't see how my initial concern is resolved or addressed here.

I have added this page:
- https://dvc-org-pr-784.herokuapp.com/doc/user-guide/data-sharing/mounted-cache

and this interactive example:
- https://katacoda.com/dvc/courses/examples/mounted-cache

that explain the case of mounted cache, which is more efficient if we share data through a NAS (with caveat of being careful with the command `dvc gc`).

Reply:
It does not answer my question unless again I'm missing the whole point of this PR. Could you please elaborate on this:

> To be precise - I don't see much value in three (two?) sections that explain different variations of the mounted remote. And to be even more precise - I haven't see a mounted share remote case. The only benefit I see - unsupported storage type. I would just create a How to or FAQ or something with one-two paragraphs explanation on options - mount, use rsync/rclone, etc.